### PR TITLE
[11.0][FIX] project.issue : add the issues unrelated to projects

### PR DESCRIPTION
**Pull request by OthmaneGhandi**
Description of the issue/feature this PR addresses: 
When migrating the issues and transform them to tasks => the issues that are not related to projects are not taking into account

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by OthmaneGhandi** [comment]
@pedrobaeza  : I have already signed the CLA => why I'm getting this message from **clabot** (I think he hates me haha :-D  )

**Comment by pedrobaeza** [comment]
About the PR, please put it on the same existing SQL, doing COALESCE where needed and putting `LEFT JOIN project_project pp ON pp.id = pi.project_id`.

About the CLA, you signed it with the email address `othmane.ghandi@gmail.com`, but this commit has been done with `othmane.ghandi@piavita.com`, so no hate, only processes working :stuck_out_tongue_closed_eyes:

**Comment by OthmaneGhandi** [comment]
@pedrobaeza I made the changes

File: addons/project/migrations/11.0.1.1/post-migration.py
Comment (pedrobaeza) [inline addons/project/migrations/11.0.1.1/post-migration.py]: Why adding extra lines inside a method? That's discouraged by PEP8 and not present in original code.

Comment (OthmaneGhandi) [reply addons/project/migrations/11.0.1.1/post-migration.py]: I didn't pay attention to it (my pep8 didn't put a warning  on it), I will remove it

**Comment by pedrobaeza** [comment]
Have you committed with the proper CLA address?

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Waiting for CLA question before merging.

**Comment by OthmaneGhandi** [comment]
No the other one (I missed it again). Should I re do it with my other account ?

**Comment by pedrobaeza** [comment]
Yes, please change the commiter email to the registered one. We can't have several email addresses under same GitHub login admitted in the CLA bot.

**Comment by OthmaneGhandi** [comment]
I changed the committer to the email that signed the CLA
Thanks @pedrobaeza

**Comment by OthmaneGhandi** [comment]
@pedrobaeza, am I missing something ?

**Comment by pedrobaeza** [comment]
Yes. You are still committing with the other email address:

Selección_028

**Comment by OthmaneGhandi** [comment]
@pedrobaeza how about this time ? 
Last time I changed the committer but this time I dropped the commit and commit it again with my email.
Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**
